python/src/spark_rapids_ml/knn.py

```diff
@@ -442,16 +457,31 @@ def exactNearestNeighborsJoin(
         return knnjoin_df
 
     def write(self) -> MLWriter:
+        """Unsupported."""
         raise NotImplementedError(
             f"{self.__class__} does not support saving/loading, just re-fit the estimator to re-create a model."
         )
 
     @classmethod
     def read(cls) -> MLReader:
+        """Uunsupported."""
         raise NotImplementedError(
             f"{cls} does not support loading/loading, just re-fit the estimator to re-create a model."
         )
 
+    def save(self, path: str) -> None:
+        """Unsupported."""
+        raise NotImplementedError(
+            "ApproximateNearestNeighbors does not support saving/loading, just re-create the estimator."
+        )
+
+    @classmethod
+    def load(cls, path: str) -> MLReader:
+        """Unsupported."""
+        raise NotImplementedError(
+            "ApproximateNearestNeighbors does not support saving/loading, just re-create the estimator."
```

Comment:
Same here, should we replace "ApproximateNearestNeighbors" with f"{cls}"?

Reply:
thanks for catching all above.